app/data_mapper_setup.rb

```diff
@@ -0,0 +1,8 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+require_relative 'models/user'
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/makersbnb_#{ENV['RACK_ENV']}")
```

Comment:
for consistency how are we naming the databases? Could they be called makers_bnb_test/development rather than makersbnb_ ?